kotsadm/pkg/handlers/troubleshoot.go

```diff
@@ -342,6 +299,7 @@ func CollectSupportBundle(w http.ResponseWriter, r *http.Request) {
 	JSON(w, 204, "")
 }
 
+// NOTE: there is no auth on this route
```

Comment:
here

Reply:
i assume this request comes from the `kubectl support-bundle` command. @laverya do you know the answer?

Reply:
Yes, that's where this request comes from
the `appId` url param might count as some form of auth, but not really secure